doc/content/modules/installation-guide/pages/how-tos/install/production_deploy.adoc

```diff
@@ -0,0 +1,102 @@
+
+This section provides instructions for installing a production-ready version of the application. This setup is intended for deployment in a live environment, with full multi-user access and all necessary security configurations.
+
+== Get {product}
+
+[CAUTION]
+.Stable versions
+====
+Only `YEAR.MONTH.0` versions are stable and suitable for production deployments.
+Other versions, such as intermediate releases, are intended solely for testing purposes.
+They're not recommended for production use due to their potential for further changes and instability.
+Always use the stable `.0` releases for any production environments to ensure reliability and support.
+For more details see xref:user-manual:cycle.adoc[].
+====
+
+== Prepare ecosystem
+
+Pay attention to deploy enterprise grade version of the prerequisites. In particular the databse service. Before do deploy manually the databse, we recomend to use managed database instance (provided by the IT service of a corporation or using cloud services)
+
+[INFO]
+====
+Before you proceed with the installation, please ensure you have the following xref:requirements.adoc[prerequisites]:
+====
+
+We recommand to provide a dedicated host (server or virtual machine) to serve the syson application and only the application.
+
+This step provide the following configuration elements :
+
+* database hostname
+* database port
+* database name
+* database username
+* database password
+
+[#download]
+== Installing {product} Manually
+
+{product} is distributed as a single _executable Java Archive_ `JAR` which contains the complete {product} application along with all its dependencies.
+This distribution simplifies the setup process, enabling convenient exploration of {product}'s features.
+
+To download the latest pre-built JAR for {product}, follow these steps:
+
+. Before you begin the installation process, navigate to the {packages-url}[download] page for `{product-jar}` to access all versions.
+. Identify the {product} latest available version for download.
++
+image::latest-version.png[Latest version]
+. Click the latest version to open its specific page.
+. In the `Assets` section, locate the JAR file named `{product-jar}`.
++
+image::download.png[Download JAR file]
+. Click the JAR file to start the download to your local machine.
+
+This process ensures that you get the most recent version of the {product} application in the form of a pre-built JAR file.
+
+[#secure-app]
+== Secure the application
+
+As explain on this xref:installation-guide:how-tos/https.adoc[guide], the {product} application must be secured to ba availlable on a network. For a production grade deployment, certificates used to secure the connexion must be valid and provided by a valid authority.
+
+Documentation to generate this kind of certificate is provided by dedicated organism as exemple (https://www.namecheap.com/support/knowledgebase/article.aspx/9422/2290/generating-a-csr-on-tomcat-using-a-keytool/)[Nameship]
```

Comment:
to generate -> for generating
kind -> type
organism -> organizations
as exemple -> such as
Nameship -> Namecheap